js/test/test.js

```diff
@@ -0,0 +1,146 @@
+const {sync, get_block_at_registration_for_all} = require("../main");
+const {expect} = require("chai");
+
+describe("Test Neurons are pulled", function() {
+        const mock_n = 1024;
+
+        const get_mock_api = (n_neurons) => {
+            return {
```

Comment:
Since this object is reused, what do you think about extracting it to use a var or const intead of copying it?

Reply:
Or maybe better extracting `get_mock_api` to a common place, right?

Reply:
they call different storage (last method in chain), but I could extract to the same API.

I did it separate as to only mock what SHOULD be used